lib/web_ui/dev/run.dart

```diff
@@ -29,13 +29,23 @@ class RunCommand extends Command<bool> with ArgUtils<bool> {
       defaultsTo: false,
       help: 'Lists all available build steps.',
     );
+    argParser.addFlag(
+      'require-skia-gold',
+      defaultsTo: false,
```

Comment:
I would default this to true. If that's not possible in this PR, then in a follow-up PR. The reason is that it's too easy to forget to set this flag, and I don't just mean when running felt manually, but in recipes and tools where we find logic that leads to false positives more often than we should. Here are some examples of past P0s and P1s that were caused because our code forgot to opt into failing tests:

- https://github.com/flutter/flutter/issues/85989
- https://github.com/flutter/flutter/issues/85387
- https://github.com/flutter/flutter/issues/78072

Reply:
But we don't want to make it true for local testing, right? Do you suggest we only enforce it in CI?